features/support/page_objects/common_page.rb

```diff
@@ -1,3 +1,12 @@
-class CommonPage < BasePage 
+class CommonPage < SitePrism::Page
   element :h1, 'h1'
```

Comment:
Should these elements not be called something meaningful ?  rather than h1, h2, p etc.. ?

Reply:
I've added this ticket for future improvements, to add data attributes with more meaningful names. https://tools.hmcts.net/jira/projects/RST/issues/RST-523?filter=allopenissues

For now I will continue to call the h1, h1 for example as it's easier to debug and it's unlikely a non technical person is going to look at page objects or step files.

Suggest a team conversation about this.